buildcontext/provider/provider.go

```diff
@@ -187,8 +156,8 @@ func (bcp *BuildContextProvider) handle(method string, stream grpc.ServerStream)
 		IncludePatterns:   includes,
 		FollowPaths:       followPaths,
 		Map:               dir.Map,
-		VerboseProgressCB: verboseProgressCB,
-	}), progress, verboseProgressCB)
+		VerboseProgressCB: progressCB.Verbose,
```

Comment:
this is weird: fsutils has both an opt and an arg.

my very quick look at this leads me to this the opt version is ignored, but I just left it as-is for now.